libglusterfs/src/inode.c

```diff
@@ -710,13 +719,17 @@ inode_new(inode_table_t *table)
 
     inode = inode_create(table);
     if (inode) {
+        if (__is_root_with_ref(inode))
```

Comment:
need to see how we can improve this - we create the root inode very very rarely. Checking it all the time for the root is somewhat inefficient. Perhaps in a different patch.

Reply:
> need to see how we can improve this - we create the root inode very very rarely. Checking it all the time for the root is somewhat inefficient. Perhaps in a different patch.

A new inode will always have 0 references. I guess we don't need to call this.

Reply:
Done.